bom-weekly/pom.xml

```diff
@@ -260,7 +260,7 @@
             <dependency>
                 <groupId>org.jenkins-ci.plugins</groupId>
                 <artifactId>mailer</artifactId>
-                <version>1.34</version>
+                <version>391.ve4a_38c1b_cf4b_</version>
```

Comment:
I assume we shouldn’t be ending with an _?

Reply:
:shrug: as in https://github.com/jenkinsci/incrementals-tools/pull/24#issue-1089971136 there are various occasions to avoid `_` without triggering the misinterpretation as `alpha` / `beta` but I did not see a need to try too hard. Should be safe to make such refinements subsequently—the algorithm used is pinned to the extension version, which is part of the source tree, so there is still a deterministic version for a given commit.

Reply:
Yeah can probably just remove it if it’s ends in it

Reply:
Feel free to refine logic after extending `MainTest.alphaBeta` to cover new cases.